packages/experimental/Shimmer/src/ShimmerTokens.android.ts

```diff
@@ -0,0 +1,14 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { ShimmerTokens } from './Shimmer.types';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+
+export const defaultShimmerTokens: TokenSettings<ShimmerTokens, Theme> = (theme: Theme) =>
+  ({
+    angle: 45,
+    delay: 0,
+    duration: 1000,
```

Comment:
Why is duration different on Android? I also noticed the animation speed is faster on Android.

Reply:
The values are based on designer guidelines for Android